packages/client/src/ee/main.model_deploy.tsx

```diff
@@ -24,6 +24,7 @@ import DeploymentEditPage from 'ee/containers/deploymentEditPage';
 import GroupSettingsPage from 'containers/groupSettingsPage';
 import GroupSettingsDeployments from 'ee/components/groupSettings/deployments';
 import GroupSettingsMLflow from 'ee/components/groupSettings/mlflow';
+import { DatasetList, DatasetDetail } from 'containers/Datasets';
```

Comment:
Should we align the import component by `import Component from '.../path'` this way?